setup.cfg

```diff
@@ -67,7 +69,9 @@ docs =
   sphinx<7.0.0
   sphinx_rtd_theme
   m2r2
+# NOTES:
 # pin sphinx because 7.0 currently not compatible with rtd theme
+# treon fails without lxml_html_clean (now a separate/optional package from lxml)
```

Comment:
`treon` doesn't include `lxml_html_clean` as a dependency? Kind of odd that you need to specify it separately